Gemfile

```diff
@@ -1,4 +1,8 @@
 source "https://rubygems.org"
+
+# Use this specific MRI ruby version.
+ruby "2.6.3"
```

Comment:
Do we need to hardcode Ruby versions? Especially for github pages that sounds like a bad idea

Reply:
I guess you are right. Let's drop that line.

Reply:
Don't forget lock :wink: